content-src/components/DiscoveryStreamComponents/List/_List.scss

```diff
@@ -88,8 +93,8 @@ $item-line-height: 20;
 // Unfortunately the border needs to go _above_ the row gap as currently
 // set up, which means that some refactoring will be required to do this.
 .ds-list-item:nth-child(-n+3) { // all but the last three items
-  border-bottom: 2px solid $grey-40-36;
-  margin-bottom: -2px;  // cancel out the 2 pixels we used for the border
+  border-bottom: 1px solid $grey-30;
```

Comment:
Same thing here: `border-bottom: $border-secondary;` Also grey-40-36 is no longer used and shouldn't exist, so remove it.

Reply:
Fixed.